pkgs/development/libraries/htmlcxx/default.nix

```diff
@@ -9,12 +9,13 @@ stdenv.mkDerivation rec {
     sha256 = "1hgmyiad3qgbpf2dvv2jygzj6jpz4dl3n8ds4nql68a4l9g2nm07";
   };
 
+  buildInputs = [ libiconv ];
```

Comment:
Is this only required for darwin?

Reply:
Yes but it's a no-op for linux since it's included with glibc.